src/Content/ForumManager.php

```diff
@@ -154,7 +154,9 @@ public static function widget(string $baseurl, int $uid, int $cid = 0)
 					'$total'	=> $total,
 					'$visible_forums' => $visible_forums,
 					'$showless'	=> DI::l10n()->t('show less'),
-					'$showmore'	=> DI::l10n()->t('show more')]
+					'$showmore'	=> DI::l10n()->t('show more'),
+					'$create_forum'	=> DI::l10n()->t('+ Create a new Forum?')
+					]
```

Comment:
I would remove the question mark, this is a button where the intent should be clear. For the plus sign, please use an icon from the forkawesome set like this: `<i class="fa fa-plus"></i>`. 
```suggestion
					'$create_forum'	=> DI::l10n()->t('Create a new Forum'),
				]
```